.github/workflows/ci.yml

```diff
@@ -0,0 +1,75 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - ci # CHANGEME
```

Comment:
Probably want to change this before merging into main, same with below

```
      - main
```

Reply:
Dan mentioned in the PR description that he will change it to main before merging